docs/dev/env_vars.rst

```diff
@@ -116,9 +116,9 @@ Disables the ``schedule`` pass.
 Set to "1", "enable", "enabled", "yes", or "true" to use.
 Disables the ``fuse_reduce`` pass.
 
-.. envvar:: MIGRAPHX_ENABLE_SPLIT_REDUCE
-Set to "1", "enable", "enabled", "yes", or "true" to use.
-Enable split_reduce.
+.. envvar:: MIGRAPHX_SPLIT_REDUCE_SIZE
+Set to the size of the reduction to do a split reduce.
+Set -1 to disable split reduce.
```

Comment:
Would it make sense to have an example of a size?  Are you talking bytes?  Suggest adding an example and what the default is now

Reply:
> Would it make sense to have an example of a size?

What do you mean example?

>  Are you talking bytes?

The number of elements, we dont ever measure the size of a reduction in terms of bytes.

> Suggest adding an example

Like `MIGRAPHX_SPLIT_REDUCE_SIZE=8192`?

> what the default is now

The default can change depending on the backend.